### ISSUE TITLE
Incorrect behavior with splits/tabs

### ISSUE THREAD

**Issue by itspriddle** [timakro/vim-searchant#3]
First, thank you for this plugin, I didn't know that I wanted this until I saw it on reddit the other day :smile:

I'm seeing some issues when using this plugin with splits/tabs.

First issue: If I have multiple split windows open and a search via `/` matches text in both splits, the other splits throw an error when navigating matches. The error (caused by [this line](https://github.com/timakro/vim-searchant/blob/6c411275c7afcebad3e8cd0986c76bf7aba5b5a8/plugin/searchant.vim#L34) in `s:Stop`):

```
Error detected while processing function <SNR>50_Update[1]..<SNR>50_Stop:
line    3:
E803: ID not found: 4
```

To duplicate, you can open `vim` with a blank file and enter some text, then run `:sp` to show the buffer in two splits. Search in one, navigate to the other and try `n` and you will see the above error. Screenshot (for clarity, I ran `/josh` in the bottom split, moved to the top with `^ww`, and pressed `n` to move to the match):

![](https://dl.dropboxusercontent.com/u/1096877/Screenshots/2016-06/Screenshot%202016-06-16%2017.38.46.png)

The second issue, possibly related: If you search for a different term in the new split, the highlighting for the current word doesn't clear in the old one in the previous split. Screenshot (here I ran `/josh` in the bottom split, moved to the top with `^ww`, and searched for `/there`):

![](https://dl.dropboxusercontent.com/u/1096877/Screenshots/2016-06/Screenshot%202016-06-16%2017.42.18.png)

Let me know if I can provide any more details!

**Comment by timakro** [comment]
Thanks for the report. The fix implemented by ed272eb is not perfect though. Unfortunately the highlighting is window specific, the plugin can't stop nor change the highlighting in the first window when you are in the second one.

**Comment by itspriddle** [comment]
Thanks!
